tests/integration/test_spm_parameterisations.py

```diff
@@ -166,7 +167,7 @@ def test_optimisers(self, optim):
         # Add sigma0 to ground truth for GaussianLogLikelihood
         if isinstance(optim.cost, pybop.GaussianLogLikelihood):
             self.ground_truth = np.concatenate(
-                (self.ground_truth, np.asarray([self.sigma0]))
+                (self.ground_truth[:2], np.asarray([self.sigma0]))
```

Comment:
I believe this is the same as the removed line?
```suggestion
                (self.ground_truth, np.asarray([self.sigma0]))
```